syntax = "proto3";

package transport_catalogue_serialize;

import "svg.proto";

message RendererSettings {
    double width = 1;
    double height = 2;
    double padding = 3;
    double line_width = 4;
    double stop_radius = 5;
    int32 stop_label_font_size = 6;
    int32 bus_label_font_size = 7;
    Point bus_label_offset = 8;
    Point stop_label_offset = 9;
    Color underlayer_color = 10;
    double underlayer_width = 11;
    repeated Color color_palette = 12;
}